src/dmd/expressionsem.d

```diff
@@ -1960,6 +1960,7 @@ private bool functionParameters(const ref Loc loc, Scope* sc,
                     arg = arg.optimize(WANTvalue, (p.storageClass & (STC.ref_ | STC.out_)) != 0);
                 }
             }
+
```

Comment:
?

Reply:
The if block should be separated from subsequent if-else chain.